app/background.js

```diff
@@ -36,6 +39,7 @@ class BackgroundWorker {
       (details) => {
         this.data[tabId].count += 1;
         chrome.browserAction.setBadgeText({text: '' + this.data[tabId].count, tabId: tabId});
```

Comment:
What I'm doing here is convert an integer into a string using the old trick `'' + integer` . Do you mean to say do this instead? : `` `${integer}` ``